Sources/MountebankSwift/Api/Mountebank+Completion.swift

```diff
@@ -288,6 +288,16 @@ extension Mountebank {
         mapAsyncToCompletion(asyncFunction: { try await getLogs(parameters: parameters) }, completion: completion)
     }
 
+    /// Validate if the Mountebank server is up with a completion handler
+    ///
+    /// Before you want to setup all your Mountebank ``Imposter``'s you can use this method to check if Mountebank
+    /// is running.
+    ///
+    /// - Throws: `MountebankValidationError` if connection between the client and server fails in some way.
```

Comment:
This does not throw any more right? It will just return an error result. Same for the other methods?